sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Serialization/SchemaRegistrySerializer.cs

```diff
@@ -500,7 +500,7 @@ private async Task<string> GetSchemaIdAsync(string schema, string schemaName, bo
             try
             {
                 // Attempt to deserialize
-                var dataStream = data.ToStream();
+                using var dataStream = data.ToStream();
```

Comment:
FWIW I think the dispose is a no-op here since the backing stream is a MemoryStream.

Reply:
I believe this is correct, but better to always dispose a disposable unless there's a good reason to not do so.

Reply:
Absolutely, I only mentioned it in case we were thinking of shipping a patch with this fix.